etl/steps/data/garden/animal_welfare/2024-12-12/farmed_finfishes_used_for_food.meta.yml

```diff
@@ -0,0 +1,85 @@
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+      - Animal Welfare
+      - Fish & Overfishing
+
+tables:
+  farmed_finfishes_used_for_food:
+    variables:
+      n_fish_low:
+        title: Estimated number of farmed finfish killed for food (lower bound)
+        unit: 'fish'
+        short_unit: ''
+        display:
+          numDecimalPlaces: 0
+        presentation:
+          grapher_config:
+            hasMapTab: false
+      n_fish:
+        title: Estimated number of farmed finfish killed for food
+        unit: 'fish'
+        short_unit: ''
+        description_short: Finfish refers to any fish with fins, as opposed to shellfish.
+        display:
+          numDecimalPlaces: 0
+        presentation:
+          grapher_config:
+            hasMapTab: false
+      n_fish_high:
+        title: Estimated number of farmed finfish killed for food (upper bound)
+        unit: 'fish'
+        short_unit: ''
+        display:
+          numDecimalPlaces: 0
+        presentation:
+          grapher_config:
+            hasMapTab: false
```

Comment:
nit: maybe worth defining this bit in `common`?

Seems like

```yaml
presentation:
  grapher_config:
    hasMapTab: true
```

is more rare.